src/Graph.ts

```diff
@@ -73,3 +80,85 @@ function isAnExit(board: Board, x: number, y: number) {
     y === lastRowIndex
   );
 }
+
+export function positionToCoordinate(position: string): {
```

Comment:
this function is unused, am I right? If so, you should remove it.

Reply:
This function will be used at the reunification of the PR #7 and #6.
I developed it to get a head start.